code/modules/mob/living/carbon/human/species/wryn.dm

```diff
@@ -62,7 +55,6 @@
 		wryn_sting.Remove(H)
 
 /* Wryn Sting Action Begin */
```

Comment:
They're basically moths now, so I feel strange keeping the code I did for the Wryn Stinger. This is something to bee considering.